API.md

```diff
@@ -259,6 +259,36 @@ If an HTTP status code of 200 is returned, the body of the response will contain
 
 If there was an error, expect an HTTP status code in either the 4XX or 5XX range.
 
+### set_inheritance
```

Comment:
can we 'see' the inheritance setting somewhere already?   in the stat information perhaps?

Reply:
oh, it's in the stat.   thanks tests!